libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -124,6 +124,7 @@
         NOTIFICATION_UNLIKED,
         NOTIFICATION_TRASHED,
         NOTIFICATION_FLAGGED_AS_SPAM,
+        NOTIFICATION_SWIPE_PAGE_CHANGED,
```

Comment:
We should bump Analytics for this new event in both `AnalyticsTrackerMixpanel` and `AnalyticsTrackerNosara`. Please   read the documentation about adding new events (or ping me in Slack) before setting a name for this event.